cli/src/build/onboarding/ui/steps/android-ci.tsx

```
@@ -0,0 +1,272 @@
+// short terminals, not a cap on every terminal: when the parent measures that
+// the comfortable body can't fit the viewport it flips the sticky `dense`
+// signal and threads `dense={true}` here, collapsing each body to the terse,
+// budget-fitting form (blank lines dropped, the per-provider setup advice
+// reduced to the first entry + a "… +N more" hint, the overwrite key list
+// tailed to the last few + a "… +N more" hint, and the target picker capped via
+// Select's `visibleOptionCount` with a "+N more" hint). `dense` defaults to
+// `false` so a component rendered without the prop (e.g. a test asserting the
+// comfortable form) gets the original look. All props/handlers/behaviour are
+// identical across both modes.
+import type { FC } from 'react'
+import type { CiSecretSetupAdvice, CiSecretTarget } from '../../ci-secrets.js'
```

ESLint

[error] 'CiSecretTarget' is defined but never used.

(unused-imports/no-unused-imports)